### PR TITLE
fix: improve landing flare (remove ground effect float)

### LINKED ISSUE #2817: [BUG] Rotation/lift off is too quick
**Mod Version**

Latest custom FBW dev verison.

**Describe the bug**

While the flight model is much better with the custom FBW, the plane "jumps" off the ground very quickly right as rotation starts. Additionally, the rotation of the nose seems too quick with too little sidestick input required. 

**To Reproduce**

1. Do a normal takeoff in the custom FBW version.
2.
3.

**Expected behavior**

Nose slowly raises and eventually, airplane lifts off.

**Actual behavior**

Nose quickly raises and airplane lifts off almost immediately. 

**References**

Comparison with FSLabs. Same conditions, weight, and CG. Note that the FSLabs requires sidestick input up to about 10 degrees to get the nose off the ground whereas the FBW starts rotating with about 5 degrees of sidestick deflection. Also note that the FBW is at 60 feet RA just when the FSLabs achieves a positive rate of climb.

https://www.youtube.com/watch?v=BWKbU_9yIuE&feature=youtu.be

**Additional context**

Was this working before/when did the issue start occurring?
Always an issue.
Is this a problem in the vanilla unmodded game?
Yes

Discord username (if different from GitHub):

### LINKED ISSUE #3092: [BUG] excessive elevator efficiency
**Mod Version**

Master FBW

**Describe the bug**

I was carry out a normal landing, with a C.G. between the forward and aft limit of %Chord at ~25%. 4.000kg fuel remain, only 23% payload. Autobrake 1 and Reverse idle, full flaps.
@ around 70-60 knots I recognise that the frontwheel do not completly touched down, so I give a ~60-80% pitch-up on the stick and the nose began to rise again! and that WITH brake on low, witch also produce a pitch down moment on the ground. 
That is quite unrealistic, that you can pitch the plane up again below around ~60-70knots. That is, to my mind, also the reason for the "excessive pitch rate" on takeoff-rotation. (another problem, that was stated here from me) 

**To Reproduce**

1.in my case, carry out a normal landing with flaps full, autobrake low, ~20% payload, 4t fuel remain. 
2.you will recognise, that the front-gear don´t really want to touch the ground
3.pull back the stick @ around 70-60 knots....you will experiance an unrealistic pitchup at that low speed.

**Expected behavior**

real plane never pitch up again @ that low speeds with CG in the limits

**Actual behavior**

too excessive elevator/stabiliser efficiency

**References**

**Additional context**

I´m a real world Airbus Pilot, and when we flare, touch down and let the stick in the neutral position, the nose really clearly touches down. We also have to pitch against it that we don´t touch too hard with the front gear. especially after the brakes apply.

by the way, yes, I have selected model to "modern" in the FS ;) 

Was this working before/when did the issue start occurring?
-no
Is this a problem in the vanilla unmodded game?
-i don´t know

Discord username (if different from GitHub):

### PR DESCRIPTION
**Pull request by donstim**
Fixes excessive float on landing due to ground effect and partially addresses too quick rotation on takeoff.

Addresses #3092, Partially addresses #2817 

## Summary of Changes

1. Removes excessive ground effect
2. Increases effect of spoiler deployment on lift to improve landing performance and fix issue of being able to pitch the nose up at 60-70 knots after landing.
3. Increase induced drag of flaps to help excessive initial climb on takeoff rotation.
4. Very slight increase in zero lift drag.
5. Incorporate Asobo's changed pitching moment due to elevator deflection table.
## Additional context

Excessive float on landing has been a longstanding issue. That is the primary issue this PR is intended to address. However, the resolution of that issue by reducing/removing the excessive ground effect should also at least partially improve the takeoff rotation performance realism.

The ground spoiler effectiveness is hampered by the inability to separate out the ground spoiler function from the in-air speedbrake function as yet. The spoiler parameters this far have largely been set to model speedbrakes. This PR increases the lift dumping aspect (to about 1/2 the effect expected on the ground so as to not be excessive in the air). Speedbrake drag has been adjusted slightly to retain (within 1-2%) the drag level of the current speedbrakes. (Although this drag effect is less than would be achieved in ground operations, the primary benefit of spoilers is to increase braking performance by increasing the weight on the wheels (by reducing lift).

A slight change to the flaps up zero lift drag level reinstates the performance level introduced by PR #2398 . At that time, the autothrust system was allowing a bit of overspeed, which helped the descent rate. Since then, an update changed the autopilot behavior to hold speed better in a 250 KCAS descent from 10,000 feet to 1,500 feet. This caused a slight reduction in descent rate. A very small change was made in the drag level to regain the intended descent rate (equivalent to about 20 seconds in a 6-minute descent).

The level of induced drag has been increased slightly for flaps down configurations to help reduce the initial climb rate of the runway a bit on takeoff rotation. It mainly affects the very high CL region (reached during the takeoff rotation transient) of the envelope with flaps down). It does increase drag by about 1% during landing, with the effect generally being larger at full landing flaps at Vapp. 

Asobo incorporated a revised pitching moment vs elevator deflection table in their last A320N update to improve precision in controlling pitch. This change also goes in the direction that I have been investigating to help with the excessive pitch on rotation issue, though it doesn't go nearly far enough. I included it in the hopes that it will also help a bit with that issue, and does not appear to have introduced any major side effects. I have held off on doing any more to address this issue because it is being addressed by the new FBW system.

**Disclaimer** This is intended as an interim fix for the current dev and stable versions while the custom fly-by-wire system is in development. It is not intended to be compared to, or incorporated in, that branch. (I am assuming that the custom fly-by-wire system already addresses the issues addressed by this PR.)

Discord username (if different from GitHub):donbikes#4084

## Testing instructions

1. The primary thing to test is landing flare performance. The airplane should no longer float excessively, and you may need to adjust your landing technique after getting used to the current developer and status versions of the mod.
2. Note whether the flare is eliminated and whether the landing is now "too firm."
3. After landing with autobrakes low and speedbrakes auto-deployed, verify that the airplane will not be able to pitch the nose up at 60-70 knots. (Note: It will still be able to do it at about 80 knots and above.) (Ref Issue #3092 )
3. Verify this PR does not introduce noticeably different descent and deceleration performance (drag) with either flaps down or speedbrakes deployed compared to the current developer version.

## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created and uploaded.
The build script will have already been run with the latest changes, so no need to rerun it once you download the zip.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the right side, click on the **Artifacts** drop down and click the **A32NX** link

### CODE REVIEW

File: A32NX/SimObjects/AirPlanes/Asobo_A320_NEO/flight_model.cfg [hunk 1]
```diff
@@ -327,7 +327,7 @@ wingflex_scalar = 0.75
 wingflex_offset = -0.25
 
 [REFERENCE SPEEDS]
-full_flaps_stall_speed = 115 ; Knots True (KTAS)
+full_flaps_stall_speed = 121 ; Knots True (KTAS)
```
Comment (MisterChocker) [inline A32NX/SimObjects/AirPlanes/Asobo_A320_NEO/flight_model.cfg:330]: why does this need to be changed?

Comment (donstim) [reply A32NX/SimObjects/AirPlanes/Asobo_A320_NEO/flight_model.cfg]: Oops, left some testing artifacts. Thanks for noticing. Also left one intended change unexplained. I will add an explanation for that one (pitch moment due to elevator movement).

**Comment by beheh** [comment]
> Partially fixes #2817

Careful, that line will fully auto-close the PR when this is merged because "fixes" is before the PR number. Try to reword it or add a comma or something so "fixes" is no longer underlined.

**Comment by MapleNAV-TCASFAULT** [comment]
Closed #2993 in favor of this

**Comment by stescouse** [comment]
Category: QA Tester
Name: Discord User stescouse#7271
Date of testing: 27.1.2021
Version of the sim: 1.12.13.0
PR Tested: #3202
Tier of testing: 3
Changes to observe: improve landing flare (remove ground effect float)

Testing technique: TryOut
Steps you've made to test the Pull Request:

1. Spawned C&D at EDDF.
2. Flew to LIRN took off and noticed heavier handling.
3. At LIRN Retarded at 30 ft above the runway, nose went right down landing rate -1121.89 ft/min.  Attempted to lift nose at 60 knots did not lift on runway.

4. Landed at EDDM.  Retarded at 20 ft height from the ground, and  landing rate of -663.6 ft/min 


https://user-images.githubusercontent.com/77486160/106016925-9f767b80-60c8-11eb-8b43-0bce6742f626.mp4



5. Landed at EDDB. Retarded at warning call on ground and landing rate of -382.17 ft/min.


https://user-images.githubusercontent.com/77486160/106016024-b9638e80-60c7-11eb-8e77-e44fef11bc17.mp4






Flight notes (If required: 3 Full flight from EDDF to LIRN, LIRN to EDDM, EDDM to EDDB.

Issues (If Any): None


Overall Rating: Good
Conclusions of the testing: Though heavier handling, once I did more flights I got used to it and landing rate improved.  A more realistic model that improves flying skills.

**Comment by donstim** [comment]
> > Partially fixes #2817
> 
> Careful, that line will fully auto-close the PR when this is merged because "fixes" is before the PR number. Try to reword it or add a comma or something so "fixes" is no longer underlined.

Is that new?  I didn't underline it. It must automatically do that now?

**Comment by Benjozork** [comment]
Whenever you have the word "Fixes" or "Closes" behind a PR tag it will auto close them on merge.

**Comment by donstim** [comment]
> Whenever you have the word "Fixes" or "Closes" behind a PR tag it will auto close them on merge.

Well, in that case I'm changing #3092 as well until and unless @andrejetcat agrees that it fixes it. Just love this system. I never seem to get it right.

**Comment by cianwhite** [comment]
Category: QA Tester
Name: Cian#3596
Date of testing: 27/01/21
Version of the sim: 1.12.13.0
PR Tested: #3202 
Tier of testing: 1/2/3
Changes to observe: Observe take-off and flare characteristics change.

Testing technique: 

- Loaded in C&D at EIDW and completed a full flight to EGGP.
- Performed circuits at EIDW with 5/6 landings.
- Tried out in a mid-air start to EIDW
- Performed take-offs and landings in the CFBW version of the mod to have somewhat of a comparison.
- Ensured I was a similar weight for all the landings for both versions (approx. 3t of fuel onboard with default MSFS payload)

Steps you've made to test the Pull Request:
1. Completed a flight from EIDW - EGPP. Starting C&D and everything went as expected. Once I came for the landing I felt the flare mechanics felt very sloppy and hard to be precise. It feels like the inputs are delayed compared to what I would expect.
2. Spawned in on an approach on EIDW 28L, the PFD isn't working as if the ADIRS aren't aligned.
3. Spawned in on the runway at EIDW and performed circuits, testing the flare mechanics and rotate mechanics, of which I observed the takeoff rotation changes.
4. On all of these landings I had the same issue as before, it feels like the inputs take some time to register and feels very sluggish.
5. Loaded in the CFBW and tried a few landings and take-offs, I felt more in control of the flare and in general everything felt more how I would expect it to on landing. As for take-offs I don't see the issue 

Flight notes (When done a full flight): LIFFY6A LIFFY Q37 MALUD TIPO3A

Media: https://streamable.com/22ek1b - Comparison between CFBW and this PR for landing mechanics (it is worth noting that this is probably the worst example of landings, I did do smooth landings however they were incredibly high in pitch, approx 7.5 degrees at 30ft and felt unrealistic).
            
image - What happens when you load in on an approach 

Issues (If Any): In my opinion the flare model seems unresponsive and sluggish, perhaps a real life pilot could correct me but I much prefer the CFBW flare mechanics and it seems to better represent how the aircraft should behave in real life. 
PFD becomes blank when spawned in on an approach (no liveries)

Steps to achieve the issue: 

- Fly some circuits and observe the mechanics of the flare
- Spawn in on an approach and observe the PFD

Overall Rating: To be improved
Conclusions of the testing: I believe the take-off mechanics work very well, much better than the current dev, with the implementation of the rotate law I have no issues listed in #2817.  I also believe that the CFBW as it is right now rotated a lot easier than this PR, leading to a more realistic rotation at approx. 3 degrees per second.

In my opinion, the flare mechanics need some work as the aircraft seems to behave sluggishly to flare input at the correct height (20-30 feet). It's also important to fix the PFD on approach, I didn't have this on runway starts so I'm unsure as to why this happened.

**Comment by BritishAvgeek** [comment]
Category: QA Tester
Name: BritishAvgeek#1436
Date of testing: 27/01/2021
Version of the sim: 1.12.13.0
PR Tested: #3202 
Tier of testing: 3
Changes to observe:
Landing Flare, Ground Float changes, Ground Spoilers changes etc

Testing technique: Full two sector return flight on VATSIM - livestreamed (see media)
Steps you've made to test the Pull Request:
1. Spawned C&D at EGCC, prepared for flight using simbrief and clearance via VATSIM. Taxiied to runway - 65T approx TOW
2. During departure, takeoff was monitored and the aircraft definitely felt 'heavier' than before during rotation, initial climb was nice too - the slight decrease in lift during rotation was definitely noticeable but very nice
3. Approach, used speedbrake during speed descent to see if it felt different - didn't notice much difference in speedbrake performance actually
4. Landed - increase drag was definitely noticeable, aircraft felt a LOT heavier and decel was improved and more noticeable too
5. Turnaround for return to EGCC
6. Departed ESGG - TOW around 67T. Noticed same as before, aircraft felt heavy and the rotation needed more input. Felt really nice
7. During descent into EGCC, activated APP Phase via perf page of MCDU - it didn't put the A/T into approach phase weirdly - it kept trying to keep me at 250kts so I had to intervene and manage the speed selection manually even on final approach
8. 

Flight notes (When done a full flight):
LEG1 - EGCC to ESGG
Take-off and initial climb felt very nice.
Landing Notes: On arrival to ESGG, flew the ILS runway 21 and had very low visibility RVR 500m or so in freezing fog. Cloud base around 500ft. On landing, idled throttle at "RETARD" and flared and found it considerably different. Landing was much harder - felt like I was flaring but the aircraft planted itself onto the tarmac. Low Autobrake was chosen but the aircraft definitely slowed down nicely and it seemed as though the ground spoilers tweaks were making a difference to "pushing the wheels into the tarmac".
Aircraft felt heavier near the ground, definitely no float on arrival!
I tried to pull back on the stick at 60-70kts, nosewheel stayed on the ground

LEG2 - ESGG to EGCC
During descent to EGCC I was in selected speed mode and was vectored in by approach. APP mode activated manually but this time it was as if the aircraft didn't recognise the input oddly. Autothrust tried to keep me at 250kts entirely even at 5DME on the ILS so selected speed was used for the remainder of approach and landing. 
Landing this time was smoother than attempt one but the observations remain the same - no float, much reduced "ground effect" and a heavier feeling to the aircraft overall.
Again on landing, I pulled back fully on the stick at 65kts and the nose remained on the tarmac!

Media: Link to the Live stream here https://youtu.be/N-Sz8jss-_M 
Relevant Times:
Takeoff 1: 14:30 onwards
Landing 1: 1:40:00 onwards
Takeoff 2: 2:18:00 onwards
Landing 2: 4:05:00 onwards

Issues (If Any): During testing of this PR my Autothrottle went haywire on the return journey during descent in the STAR and ILS approach... not replicated any other time but I'll monitor and check the dev version another time. Otherwise no issues.
Steps to achieve the issue N/A

Overall Rating: Good
Conclusions of the testing: The aircraft felt like I was rotating 65T into the air and the wider feel during flare, touchdown, rollout felt heavier and more intentional. It definitely will require re-adjustment for landing but I really like these changes! 

Speedbrake in descent hasn't been adversely affected, and flap settings do produce some additional drag which is great. The slight increase in drag is certainly noticeable towards the last 50ft too - so idling the throttle before "retard" would now likely smack you down hard! Great changes

**Comment by donstim** [comment]
I want to thank the QA testers for their time and their comments.

**Pull request closed without merging**